openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -957,7 +957,7 @@ To validate a key proof, the Credential Issuer MUST ensure that:
 - the header parameter indicates a registered asymmetric digital signature algorithm, `alg` parameter value is not `none`, is supported by the application, and is acceptable per local policy,
 - the signature on the key proof verifies with the public key contained in the header parameter,
 - the header parameter does not contain a private key,
-- the `nonce` claim (or Claim Key 10) matches the server-provided `c_nonce` value, if the server had previously provided a `c_nonce`,
+- if the server had previously provided a `c_nonce`, the nonce in the key proof matches the server-provided `c_nonce` value,
```

Comment:
no need for `(or Claim Key 10)` :)?

Reply:
This part was broken as well because it didn't include `challenge` for ldp proofs. I just fixed the language by generalizing the nonce for all proof types. How nonce is done is described in the proof type sections. Does this work for you?